javascript/triage_bot/app.js

```diff
@@ -0,0 +1,76 @@
+const asana = require("asana");
+const config = require("./config");
+
+// Get personal access token (PAT) from environment variables.
+const accessToken = process.env.triage_bot_pat;
+
+// At the time of writing this guide, the Asana API is going through 
+// two deprecations (moving to string ids and changing how sections function).
+// You can learn about our deprecations framework in our docs
+// (https://asana.com/developers/documentation/getting-started/deprecations).
+// To prevent my app from breaking when the deprecations are finalized, I'm 
+// passing headers to enable the new API behavior for string gids and sections.
+const deprecationHeaders = {"defaultHeaders": {"asana-enable": "new_sections,string_ids"}};
+
+// Create Asana client using PAT and deprecation headers.
+const client = asana.Client.create(deprecationHeaders).useAccessToken(accessToken);
+
+
+// Request the unassigned tasks (up to 100) from the specified project
+// and pass the array of unassinged tasks to randomAissinger().
+function getUnassingedTasks() {
+  let workspaceId = config.workspaceId;
+  let params = {
+    "projects.any" : config.designRequestProjectId,
+    "assignee.any" : "null",
+    "resource_subtype": "default_task",
+    "fields": "gid",
+    "limit": 100
```

Comment:
Do we want to add completed: false ? Just to fit the narrative. Maybe not thought.